### ISSUE TITLE
Email notifications?

### ISSUE THREAD

**Issue by jipiboily** [pyupio/statuspage#31]
Hey,
love what you did! I'm not using it yet but plan on doing it very soon. One thing that is missing IMHO is the ability to subscribe via email to the status updates. I know this is hosted on GitHub pages, so this is a bit tricky. Here is an idea I had. It's a bit hacky, I must admit, and totally incomplete.

- Have an RSS feed of the incidents
- Have a MailChimp account, or something similar, and integrate it with statuspage
- Have _something_ that sends email for new RSS entries

I looked at the MailChimp "RSS to email" feature, but it's not fast enough, it would be a day late, that's useless. It doesn't look like it's possible with IFTTT or Zapier, but I might have missed something.

It would be easy to host a service doing that, but I would prefer not to have to host an app to do that, but have something easy.

Maybe there is nothing that fits the vision you have, but I thought I would share my thoughts and maybe someone can complete those and we can come up with a decent solution :)

Cheers

**Comment by jayfk** [comment]
> It would be easy to host a service doing that, but I would prefer not to have to host an app to do that, but have something easy.

That's exactly what I'm thinking. The main idea behind this project is to make it super easy to set up and maintain a status page. 

If someone comes up with a clever idea to set up a notification system I'm all ears for it.

**Comment by jipiboily** [comment]
Yeah, same. I actually started to write thinking that MailChimp would work, but before sending it, I double checked, and the fact it is not in near real-time is a killer, and totally agree this should be mad simple to setup and use. Feel free to close in the meantime.
